docs/usage.rst

```diff
@@ -765,3 +709,30 @@ of relevance, are:
    though.
 
 .. _Google Base XML feed: https://support.google.com/merchants/answer/160589?hl=en&ref_topic=2473799
+.. _requests: http://www.python-requests.org/
+
+
+Similar libraries
+=================
+
+
+ * `BeautifulSoup`_ is a very popular screen scraping library among Python
+   programmers which constructs a Python object based on the structure of the
+   HTML code and also deals with bad markup reasonably well, but it has one
+   drawback: it's slow.
```

Comment:
Right, I'm not sure if it still makes sense to say that BS is slow.
Maybe we can just avoid the `drawback` bit and let our users decide. :)